CharacterCreation/Class-Specific Documentation/Engineer Processes.txt

```diff
@@ -380,91 +394,88 @@ normalize down to their maximum after ten turns.
+Material Cost:  15 Material
 Action:			2 Actions
 Duration:		Immediate
 Range:			Touch
 
-    	An adhesive mist of refractive particles that absorb laser and
-plasma fire quite well. Toxic, but slow-acting to most biologic life forms. Can
-only be used on cybernetic allies or terrain. An engineer only carries four 
-doses per engagement. Reduces the next plasma/laser/CS hit by 35 points of 
-damage and prevents the first damage-over-time effect from that hit, if it has
-one. Looks kinda like sticky glitter. Gives an ally affected by Refractory 
-spray a -20 penalty to visual stealth rolls. Refractory Spray can be detected
-by metal detectors. Refractory Spray loses cohesion after ten turns or burns up
-if hit by an energy attack.
+    An adhesive mist of refractive particles that absorb laser and plasma fire 
+quite well. Reduces the next plasma/laser/VAE hit by 35 points of damage and 
+prevents the first damage-over-time effect from that hit, if it has one. Looks 
+kinda like sticky glitter. Gives an ally affected by Refractory spray a -20 
+penalty to visual stealth rolls. Refractory Spray can be detected by metal 
+detectors. Refractory Spray loses cohesion after ten turns or burns up if hit by 
+an energy attack.
 
 == Tune Laser	== 
 Energy Gain:	30 Energy
 Action:			2 Actions
 Duration:		5 Turns
 Range:			Touch
 
-    	You dial in the potentiometers and realign the focusing 
-crystals. Your ally's weapon deals an extra 20 points of damage per hit
-and has a -10% to miss (To a maximum of 0% when target is in light cover). 
+    You dial in the potentiometers and realign the focusing crystals. Your 
+ally's laser weapon deals an extra 20 points of damage per hit and has a -1 to 
+miss (To a minimum to-miss of -0 when target is in light cover). 
 
 == MoCap Augmentation	== 
 Energy Gain:	20 Energy
 Action:			2 Actions
 Duration:		Immediate
 Range:			Touch
 
-    	By temporarily reducing the resolution of your cybernetic 
-ally's visual sensors you radically reduce the processing power it takes to
-capture motion, letting them process more frames per microsecond. This gives
-them slightly faster reaction times, resulting in a -8% chance to miss when
-attacking, but makes it much harder to see long distances. Allies with MoCap
-Augmentation active take a -40 to perception checks at ranges beyond 50m. 
-Since this is less than ideal in non-combat scenarios, you've set it to wear
-off after ten turns.
+    By temporarily reducing the resolution of your cybernetic ally's visual 
+sensors you radically reduce the processing power it takes to capture motion, 
+letting them process more frames per microsecond. This gives them slightly 
+faster reaction times, resulting in a -1 chance to miss when attacking, but 
+makes it much harder to see long distances. Allies with MoCap Augmentation 
+active take a -40 to perception checks at ranges beyond 50m. Since this is less 
+than ideal in non-combat scenarios, you've set it to wear off after ten turns.
```

Comment:
Just for aesthetic, we may want to include a more explicit reason for it being correct to let the effect wear off. Just 'less than ideal' won't stop the technical stat-optimizer players.

Reply:
agreed.